nextflow.config

```diff
@@ -30,8 +30,12 @@ params {
     skip_trimming                       = false
 
     // Cell barcode options
-    whitelist                           = null
+    cdna_whitelist                      = null
+    dna_whitelist                       = null
     barcode_format                      = null
+    custom_flexiplex_barcode_dna        = null
+    custom_flexiplex_barcode_cdna       = null
+    demux_tool                          = 'flexiplex' // Options: flexiplex, blaze
```

Comment:
I know there's only one option for dna, but may be good to go ahead and implement a demux_tool_cdna and demux_tool_dna if there are going to be different subsets for each type. But this is dependent on if BLAZE can run on teh dna samples or not